x/cellarfees/types/params.go

```diff
@@ -1,10 +1,30 @@
 package types
 
 import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	DefaultFeeAccrualAuctionThreshold uint64 = 2
+	// Rough number of blocks in 28 days, the time it takes to unbond
+	DefaultRewardEmissionPeriod uint64 = 403200
+	// Initial rate at which an auction should decrease the price of the relevant coin from it's starting price
+	DefaultInitialPriceDecreaseRate uint64 = 347000000000000
```

Comment:
It might be helpful to describe what these units mean. I guess we can't have a const `Dec` type?

Reply:
Yeah `Dec` doesn't support const. will add context.